src/constants.ts

```diff
@@ -84,6 +84,10 @@ export interface S3PluginOptions extends PluginOptions {
     // but could be useful for preventing Cloud formation Stack Drift or suppressing Terraform noise if you don't need
     // the static website hosting functionality.
     enableS3StaticWebsiteHosting?: boolean;
+
+    // Max number of files to upload in parallel.
+    // This value should not exceed the maximum number of open files supported by the system.
```

Comment:
How do we find the maximum number of open files supported by the system? Is this a limit users are likely to run into? My instinct says you're likely to hit significant network bottlenecks before this becomes a problem.

I'm happy to simplify the comment to just "Max number of files to upload in parallel", the second line seems to me to introduce more questions than it answers.

Reply:
We ran into the higher limit while testing but as you say it is likely network bottlenecks would be encountered before hitting this limit. We are happy to remove the disclaimer as we agree it is unlikely this limit will be reached.

Reply:
Please do, thanks.